x/gamm/keeper/pool.go

```diff
@@ -93,17 +93,6 @@ func (k Keeper) SetPool(ctx sdk.Context, pool types.PoolI) error {
 	return nil
 }
 
-func (k Keeper) DeletePool(ctx sdk.Context, poolId uint64) error {
```

Comment:
Seems that `DeletePool` is called by `CleanupBalancerPool` in #1927

Just linking it for when we need to bring back `CleanupBalancerPool` with the new SDK